demo/wazuh_template.yml

```diff
@@ -728,7 +758,7 @@ Resources:
               group: root
             /tmp/wazuh_cf_bootstrap_elastic.sh:
               source: !Sub >-
-                https://raw.githubusercontent.com/wazuh/wazuh-cloudformation/v3.13.0_7.7.1/demo/elastic/wazuh_cf_bootstrap_elastic.sh
+                https://raw.githubusercontent.com/wazuh/wazuh-cloudformation/devel/demo/elastic/wazuh_cf_bootstrap_elastic.sh
```

Comment:
I think this should be `master` branch.

Reply:
Replaced on 0d37757edec3d9c3f8a1ac1c23d7c49a2765bad4.